src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/skjema/SkjemaInntektsmelding.kt

```diff
@@ -15,5 +14,4 @@ data class SkjemaInntektsmelding(
     val agp: Arbeidsgiverperiode?,
     val inntekt: Inntekt?,
     val refusjon: Refusjon?,
-    val aarsakInnsending: AarsakInnsending,
```

Comment:
Trenger vi ikke dette feltet innover i verdikjeden? Vi videresender vel "ny" / "endring"

Reply:
Vi trenger det ikke for selvbestemt, som dette skjemaet brukes til i første omgang. Det finner vi ut av selv i backenden. For forespurt IM så er jeg ikke sikker, men jeg tror det også kan fjernes der, hvis vi ser at forespørselen allerede er besvart.

Reply:
OK, fair nuff